cron/daily-message.js

```diff
@@ -9,19 +9,19 @@ function createMessage({dateString, waitingFor}) {
   const releaseDate = moment(dateString).add(4, 'days');
   const hoursUntilRelease = releaseDate.diff(moment(), 'hours');
 
-  if (hoursUntilRelease >= 262 && hoursUntilRelease <= 266) {
```

Comment:
I wanted to create a function so that, when we later return to this code, we won't be confused by the numbers 262, 266, 94, 98, etc. I think it would take a while to see the pattern.